### PR TITLE
test: improve coverage

### PR DESCRIPTION
**Pull request by lubtd**
Improve coverage of the different tests

- Test untested methods
- Add new test cases
- Fix trigger launch and revert launch messages tests
- Assert errors for `keeper/msg_...` tests

### CODE REVIEW

**Comment by lubtd** [comment]
@Pantani I fixed what I observed. I let you add other improvements in this PR if you spot few

File: x/profile/keeper/coordinator_test.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package keeper_test
 
 import (
+	"github.com/tendermint/spn/testutil/sample"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
```
Comment (Pantani) [inline x/profile/keeper/coordinator_test.go:7]:
```suggestion
	"testing"

	sdk "github.com/cosmos/cosmos-sdk/types"
	"github.com/tendermint/spn/testutil/sample"
```

File: x/profile/keeper/coordinator_by_address_test.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package keeper_test
 
 import (
 	"fmt"
+	"github.com/tendermint/spn/testutil/sample"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
```
Comment (Pantani) [inline x/profile/keeper/coordinator_by_address_test.go:8]:
```suggestion
	"fmt"
	"testing"

	sdk "github.com/cosmos/cosmos-sdk/types"
	"github.com/tendermint/spn/testutil/sample"
```

File: x/launch/keeper/msg_server_trigger_launch_test.go [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package keeper_test
 
 import (
+	profiletypes "github.com/tendermint/spn/x/profile/types"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
```
Comment (Pantani) [inline x/launch/keeper/msg_server_trigger_launch_test.go:7]:
```suggestion
	"testing"

	sdk "github.com/cosmos/cosmos-sdk/types"
	profiletypes "github.com/tendermint/spn/x/profile/types"
```

File: x/launch/keeper/msg_server_revert_launch_test.go [hunk 4]
```diff
@@ -1,6 +1,7 @@
 package keeper_test
 
 import (
+	profiletypes "github.com/tendermint/spn/x/profile/types"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
```
Comment (Pantani) [inline x/launch/keeper/msg_server_revert_launch_test.go:7]:
```suggestion
	"testing"

	sdk "github.com/cosmos/cosmos-sdk/types"
	profiletypes "github.com/tendermint/spn/x/profile/types"
```

File: x/launch/keeper/msg_server_create_chain_test.go [hunk 5]
```diff
@@ -1,6 +1,7 @@
 package keeper_test
 
 import (
+	profiletypes "github.com/tendermint/spn/x/profile/types"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
```
Comment (Pantani) [inline x/launch/keeper/msg_server_create_chain_test.go:7]:
```suggestion
	"testing"

	sdk "github.com/cosmos/cosmos-sdk/types"
	profiletypes "github.com/tendermint/spn/x/profile/types"
```

File: x/launch/keeper/grpc_query_params_test.go [hunk 6]
```diff
@@ -0,0 +1,22 @@
+package keeper_test
+
+import (
+	"github.com/tendermint/spn/testutil/sample"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	testkeeper "github.com/tendermint/spn/testutil/keeper"
+	"github.com/tendermint/spn/x/launch/types"
```
Comment (Pantani) [inline x/launch/keeper/grpc_query_params_test.go:10]:
```suggestion
	"testing"

	sdk "github.com/cosmos/cosmos-sdk/types"
	"github.com/stretchr/testify/require"
	testkeeper "github.com/tendermint/spn/testutil/keeper"
	"github.com/tendermint/spn/testutil/sample"
	"github.com/tendermint/spn/x/launch/types"
```

**Review by Pantani — CHANGES_REQUESTED** [review]

File: x/launch/keeper/msg_server_edit_chain_test.go [hunk 7]
```diff
@@ -1,6 +1,7 @@
 package keeper_test
 
 import (
+	profiletypes "github.com/tendermint/spn/x/profile/types"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
```
Comment (Pantani) [inline x/launch/keeper/msg_server_edit_chain_test.go:7]:
```suggestion
	"testing"

	sdk "github.com/cosmos/cosmos-sdk/types"
	profiletypes "github.com/tendermint/spn/x/profile/types"
```

**Review by ilgooz — DISMISSED** [review]

**Review by ilgooz — DISMISSED** [review]

**Review by Pantani — APPROVED** [review]

**Review by Pantani — APPROVED** [review]

**Pull request merged**